### PR TITLE
Releasing CIRCL v1.3.6

### PR DESCRIPTION
**Pull request by armfazh**
Updating readme with new algorithms.
Bumping to v1.3.6 to jump branch tags.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,60 +25,119 @@ You can get CIRCL by fetching:
 go get -u github.com/cloudflare/circl
 ```
 
+Alternatively, you can use CIRCL natively by using [Cloudflare Go](https://github.com/cloudflare/go/tree/cf) fork.
```
Comment (bwesterb) [inline README.md:28]: It doesn't contain all packages, does it?

File: README.md [hunk 2]
```diff
@@ -25,60 +25,119 @@ You can get CIRCL by fetching:
...
+
+ - [CSIDH](./dh/csidh): Post-Quantum Commutative Group Action ([CSIDH](https://csidh.isogeny.org/)).
+ - [Kyber KEM](./kem/kyber): modes 512, 768, 1024 ([KYBER](https://pq-crystals.org/kyber/)).
+ - [FrodoKEM](./kem/frodo): modes 640-SHAKE. ([FrodoKEM](https://frodokem.org/))
+ - (**insecure, deprecated**) ~~[SIDH/SIKE](./kem/sike)~~: Supersingular Key Encapsulation with primes p434, p503, p751 ([SIKE](https://sike.org/)).
+
+| PKE: Public-Key Encryption |
+|:---:|
+
+ - [Kyber PKE](./pke/kyber): modes 512, 768, 1024 ([Kyber](https://pq-crystals.org/kyber/)).
```
Comment (bwesterb) [inline README.md:91]: We shouldn't list this, or add a warning that this is not secure for general purpose.

**Review by bwesterb — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (armfazh) [reply README.md:28]: initially, we cherry picked packages.
the latest configuration uses an explicit require in go.mod file. so the compiler resolves the set of packages used.

File: README.md (hunk 2 above)
Comment (armfazh) [reply README.md:91]: ok, it's better removing it.

File: README.md [hunk 3]
```diff
@@ -25,60 +25,119 @@ You can get CIRCL by fetching:
...
- - [DLEQ](./zk/dleq): Prove knowledge of the Discrete Logarithm Equality.
+| XOF: eXtendable Output Functions |
+|:---:|
+
+ - [SHAKE128 and SHAKE256](./xof) ([FIPS 202]).
+ - [BLAKE2X](./xof): BLAKE2XB and BLAKE2XS ([Blake2x](https://www.blake2.net/blake2x.pdf))
+ - [KangarooTwelve](./xof/k12): fast hashing based on Keccak-p. ([KangarooTwelve](https://keccak.team/kangarootwelve.html)).
+ - SIMD [Keccak](https://keccak.team/keccak_specs_summary.html) f1600 Permutation.
+
+| LWC: Lightweight Cyptography |
```
Comment (SulemanAhmadd) [inline README.md:114]:
```suggestion
| LWC: Lightweight Cryptography |
```

File: README.md (hunk 1 above)
Comment (bwesterb) [reply README.md:28]: Check the `src/vendor/github.com/cloudflare/circl` folder of cfgo. It doesn't contain many packages such as `math/fp448` and such as they aren't used by cfgo itself.

File: README.md [hunk 4]
```diff
@@ -25,60 +25,119 @@ You can get CIRCL by fetching:
...
+ - [OT](./ot/simot): Simplest Oblivious Transfer ([ia.cr/2015/267]).
+ - [Threshold RSA](./tss/rsa) Signatures ([Shoup Eurocrypt 2000](https://www.iacr.org/archive/eurocrypt2000/1807/18070209-new.pdf)).
+
+### Post-Quantum Cryptography
+
+| KEM: Key Encapsulation Methods |
+|:---:|
+
+ - [CSIDH](./dh/csidh): Post-Quantum Commutative Group Action ([CSIDH](https://csidh.isogeny.org/)).
+ - [Kyber KEM](./kem/kyber): modes 512, 768, 1024 ([KYBER](https://pq-crystals.org/kyber/)).
```
Comment (SulemanAhmadd) [inline README.md:84]: Should we name it to ML-KEM based on NIST's renaming?

**Review by SulemanAhmadd — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (bwesterb) [reply README.md:84]: It's not the same as the draft of ML-KEM. I think we might just add the final version of ML-KEM as a separate package, so we don't break early adopters (such as ourselves.)

**Pull request merged**